ui/models/job.js

```diff
@@ -90,7 +90,11 @@ export default class JobModel {
       notify(`Attempting to retrigger/add ${jobTerm} via actions.json`, 'info');
 
       const pushIds = [...new Set(jobs.map(job => job.push_id))];
-      const taskIdMap = await PushModel.getDecisionTaskMap(pushIds, notify);
+      const taskIdMap = await PushModel.getDecisionTaskMap(
```

Comment:
This function in the ``PushViewSet`` actually ignores the ``project`` param (which is the same as ``repoName``).  So you don't need to pass it here.